BagelzModels/OrderViewModel.cs

```diff
@@ -19,6 +19,11 @@ public class OrderViewModel
 
         public virtual List<OrderItemViewModel> OrderItems { get; set; } = new();
 
+        public decimal TaxRate { get; set; } = 0.08m;
```

Comment:
This should either have [JsonIgnore] or be applied to the model. Applying to the real model is probably better. It would require updates to the migrations, but would allow the tax to change in the future while still reporting accurate costs for past orders.